force-app/main/default/pages/RelationshipModelGenerator.page

```diff
@@ -36,6 +36,10 @@
                     <apex:outputLabel value="Relationship Fields to Exclude"/>
                     <apex:inputTextarea value="{!exclusions}" rows="5" style="width: 30em"/>
                 </apex:pageBlockSectionItem>
+                <apex:pageBlockSectionItem >
+                    <apex:outputLabel value="Extensible Model Classes"/>
+                    <apex:inputTextarea value="{!extensibleModelClasses}" rows="3" style="width: 30em"/>
+                </apex:pageBlockSectionItem>
```

Comment:
Instead of the class names, let the user set the SOB types like above (i.e. field `Make Model Classes extensible for Types` vs `Extensible Model Classes`.

Also, provide an help text of "New-line separated list of SObject Types. The relative Wrapper classes will be `virtual`, so extensible." (with the `pageBlockSectionItem` attribute `helpText="New-line separated list of SObject Types. The relative Wrapper classes will be `virtual`, so extensible."`